silabs-multiprotocol/Dockerfile

```
@@ -165,11 +165,7 @@ ENV BACKBONE_ROUTER=1
 ENV WEB_GUI=1
 ENV DOCKER 1
 
-COPY otbr-patches/0001-Avoid-writing-to-system-console.patch /usr/src
-COPY otbr-patches/0001-rest-support-erasing-all-persistent-info-1908.patch /usr/src
-COPY otbr-patches/0002-rest-support-deleting-the-dataset.patch /usr/src
-COPY otbr-patches/0003-mdns-update-mDNSResponder-to-1790.80.10.patch /usr/src
-COPY otbr-patches/0004-mdns-add-Linux-specific-patches.patch /usr/src
+COPY otbr-patches/0001-PATCH-rest-Support-deleting-the-dataset.patch/usr/src
```

Hadolint (2.12.0)

[error] unexpected end of line. At least two arguments are required for COPY

(DL1000)